src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -84,6 +80,11 @@ def json_error_page(status, message, traceback, version):
                            version=version))
 
 
+_cp_config_default = {
```

Comment:
Why not let it part of the BaseControllerMeta?

Reply:
`AuthRequired()` can now be used independent of `BaseController`